unbound/USAGE-client.md

```diff
@@ -0,0 +1,62 @@
+
+# Usage of the client side container
+
+The post quantum key exchange [openssl](https://github.com/open-quantum-safe/openssl) variant and [getdns](https://getdnsapi.net/) were installed to query the DNS server with a post quantum key exchange algorithm.
+## Installation
+Assuming you have docker [installed](https://docs.docker.com/install) on your machine all command below will launch dns server docker.
+
+Run getdns container:
+```bash
+    cd getdns-docker && \
+    docker build -t getdns:dev . && \
+    docker run --interactive --tty --hostname getdns --name getdns getdns:dev
+```
+After running all the command above a container will open with getdns running with openssl post quantum variant.
+
+## Usage
+
+In the file [Dockerfile](getdns-docker/Dockerfile#L45) the environment variable  TLS_DEFAULT_GROUPS is set "p384_kyber768:X25519" to force for p384_kyber768 key exchange. Other key exchange algorithms can be used, find more algorithm in the [list of available post quantum key exchange algorithms](https://github.com/open-quantum-safe/boringssl#key-exchange). 
+
+```bash
+    ENV TLS_DEFAULT_GROUPS="p384_kyber768:X25519"
+```
+Or in the interactive container the environment variable can be set with desired key exchange algorithm from the [list](https://github.com/open-quantum-safe/boringssl#key-exchange):
+```bash
+    export TLS_DEFAULT_GROUPS="p384_kyber768:X25519"
+```
+
+To query the DNS server, run the command:
+
+```bash
+    getdns_query -s -d example.com A @<DNS server container ip>:853 -L +return_call_reporting
+```
+
+By default the DNS server container and the getdns container are connected on the bridge network.
+To get the ip address of DNS server container use the command and check the "unbound_pro" ip address:
+
+```bash
+    docker network inspect bridge
+```
+Below is an example output of the command where "unbound_pro" has the ip address of 172.17.0.3.
```

Comment:
Wouldn't it be easier to run this within a [docker network](https://docs.docker.com/network/)?

Reply:
I used a network for it but the <getdns_query> does not convert the container tag to its ip address, that why i show how to get the ip address of the dns server container.